Atlas Edit Proposal — 2026-06-08 (#255)
makerdao/next-gen-atlas: content/A/1/12/2/1/7/1/document.md

###### A.1.12.2.1.7.1 - Procedure For Blocking AEP For Misalignment [Core]

An Atlas Edit Proposal can be blocked by the Core Facilitator for misalignment.

Where an AEP has been blocked for misalignment, the Core Facilitator must update the AEP status as "Rejected-Misaligned".

> The Core Facilitator should issue a written advisory opinion containing their findings/reasoning for blocking the AEP.

Reviewer: Content analysis of amendments regarding logical consistency, systemic loopholes, and linguistics: [Notion](https://sky-proposal-analysis.notion.site/Atlas-Edit-Weekly-Cycle-Proposal-2026-06-15-37ea66bbb6b8802281e5cfa68ee5a1a8).

```suggestion
The Core Facilitator must issue a written advisory opinion containing their explicit technical reasoning for blocking the AEP.
```
